chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/DeveloperStatus.java

```diff
@@ -22,12 +22,12 @@ public class DeveloperStatus implements Serializable {
     private Long id;
 
     @Schema(description = "Developer status name")
-    private String status;
+    private String name;
 
     public DeveloperStatus(DeveloperStatus other) {
         this();
         this.setId(other.getId());
-        this.setStatus(other.getStatus());
+        this.setName(other.getName());
     }
 
     @Override
```

Comment:
Is there something "special" about these equals and hashcode methods?  Can we just use the standard Lombok ones that are implemented with `@Data`?